src/backend/api/meals.js

```diff
@@ -1,15 +1,65 @@
 const express = require("express");
 const router = express.Router();
 const knex = require("../database");
+router.use(express.json())
 
-router.get("/", async (request, response) => {
+
+router.get("/", async (req, res) => {
+  try{
+    const getAllMeals = await knex('meal').select('*')
+    res.json(getAllMeals)
+    console.log(getAllMeals);
+  } catch(err){
+    throw(err)
+  }
+})
+
+router.post("/", async (req, res) => {
+  try {
+    const insertMeal = await knex('meal').insert({ id: req.body.id, title: req.body.title, description: req.body.description, location: req.body.location, when_date: req.body.when_date, max_reservations: req.body.max_reservations, price: req.body.price, created_date: req.body.created_date});
```

Comment:
It should not be possible to set `id` and `created_date` as those are automatically set.